spec/openapi.yaml

```diff
@@ -112,30 +117,35 @@ paths:
             schema:
               type: object
               properties:
+                # platform dependent
                 command:
                   type: string
                   default: connect
+                # platform dependent
                 baudrate:
                   type: integer
                   default: 115200
+                # platform dependent
                 port:
                   type: string
                   default: VIRTUAL
+                # platform dependent
                 printerProfile:
                   type: string
                   default: _default
+                # platform dependent
                 connect:
                   type: object
                   properties:
                     hostname:
                       type: string
-                      example: "dev.connect.prusa"
+                      example: "connect.prusa3d.com"
                     port:
                       type: integer
-                      default: 8080
+                      default: 0
                     tls:
-                      type: boolean
-                      default: false
+                      type: integer
+                      default: 1
```

Comment:
why don't you want to have it as a boolean?

Reply:
Fixed back to boolean